src/mkdir.cpp

```diff
@@ -0,0 +1,29 @@
+#include "commands.h"
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include <cctype>
+#include <iostream>
+void sub1(const std::string & input);
```

Comment:
This should be a lambda inside `mkdir` or be a `static` function. Either way, forward declaration is less maintainable than simple declaration.

Reply:
how would I specifically avoid forward declaration